### ISSUE TITLE
"Won't fix" issues (to be updated)

### ISSUE THREAD

**Issue by qoqobolo** [status-im/status-react#11271]
* [ ] **ISSUE 1 - If shift the focus to the beginning of the amount on the Send transaction view and try to correct it (e.g. delete first char), new chars will be shifted off the screen.**

**Operating System:** iOS 

![ezgif com-gif-maker (4)](https://user-images.githubusercontent.com/67952253/94944068-ff716280-04d8-11eb-888c-2df07666f855.gif)

- - -
* [ ] **ISSUE 2: Animation of the "Copied" message is broken when copy the invite link to the group chat with a long press.**

_Steps:_

* Create group chat
* Open chat settings > Group info
* Tap Group invite
* Long tap Grop invite link

**Operating System:** Android, iOS 

![ezgif com-gif-maker (5)](https://user-images.githubusercontent.com/67952253/94945355-d6ea6800-04da-11eb-84ed-839c08708f54.gif)

- - -
* [ ] **ISSUE 3: If use the gesture to swipe the screen to go back during onboarding, then when the user goes to the next screens, those screens from which the User went back will be skipped.**

E.g. if the user swipes back from the password generation screen to the choosing 3-random name screen and then tap "Next", password screen will be skipped, User won't be able to enter and save it. So at the end, the User could be inside the new account, without a selected password, but it’s required anyway.

_Steps:_

* Instal Status
* Tap Get started
* Tap Generate keys
* Swipe back via the gesture 
* Tap Generate keys
* Swipe back
* Tap Generate keys

**Operating System:** iOS 

![ezgif com-gif-maker (8)](https://user-images.githubusercontent.com/67952253/95012275-a53ae380-0637-11eb-9303-0480d111a1f5.gif)

- - -
* [ ] **ISSUE 4: If a token was added only to the main account, on the Watch-only account it will be shown as "...WEENUS / ...USD" until relogin.**

**Operating System:** Android

![Screenshot 2020-10-08 at 17 18 51](https://user-images.githubusercontent.com/67952253/95478838-591ad680-098a-11eb-8802-9743d25efe77.png)

- - -
* [ ] **ISSUE 5: If block some user in public chat, and then unblock him, his name will be shown in the mention suggestions list in 1-1 and group chats with other users.**

#### Expected behavior
Mention suggestions list contains contacts and doesn't contain users from other chats, blocked, and unblocked users.

_Steps:_

* Join public chat
* Block some user and then unblock him
* Start 1-1 chat with another user
* Type @ 
* Check the suggestions list

* [ ] **ISSUE 6: `Fetch more messages` button is still in the chat view after `Use history node` has been disabled, and when pressed, an endless spinner appears**

#### Expected behavior
No `Fetch more messages` button, if `Use history node` is disabled; no spinner.

_Steps:_

* Disable `Use history node`
* Join pub chat (expected = actual = no `Fetch more messages` button, no history)
* Enable `Use history node`
* Open pub chat (expected = actual = history was fetched)
* disable `Use history node`
* Open pub chat
* Tap `Fetch more messages`

![ezgif com-gif-maker (6)](https://user-images.githubusercontent.com/67952253/99247822-c5012200-2807-11eb-80f8-140a5c961b52.gif)

- - -
* [ ] **ISSUE 7: If User follows the invite link to a group chat, sends a request to join, the admin accepts it, but User declined the invitation, he can follow the same link again, and send request a second time. But after the second request is accepted, User, in fact, will not be able to join the group chat after tapping "Join" (sent messages from other members will not be received by this User)**

#### Expected behavior

* Сan join a group chat via the same Invite link even if the first invitation has been declined
* Or can't use the same invitation link

_Steps:_

**_User A_**:

* Create a group chat
* Send Invite link to User B

**_User B_**:

* Follow the invite link
* Send request to join

**_User A_**:

* Accept request 

**_User B_**:

* Tap Decline in the system message in group chat
* Follow the same invite link again
* Send request to join

**_User A_**:

* Accept request 

**_User B_**:

* Tap Join

**_User A_**:

* Send some messages to the group

**_User B_**:

* [ ] Check chat history
* [ ] **ISSUE 8: Blank screen after deleting the last multi-account**

#### Expected behavior
"Truly private communication" image on the screen
![](https://user-images.githubusercontent.com/67952253/100625690-20adce00-3325-11eb-95f1-af3270de4696.png)

#### Actual behavior
Blank screen
![ezgif com-gif-maker (9)](https://user-images.githubusercontent.com/67952253/100625957-77b3a300-3325-11eb-8fa3-c5d1bfb645c1.gif)

- - -
* [ ] **ISSUE 9: "Unknown" name in the membership request to group chat instead of 3-random name**

_Steps:_

User A:

* Create a group chat
* Open group info > Group invite > Send group invite link to any chat
User B:
* Follow the group invite link
* Send a membership request
User A:
* Open the group chat
* Tap "Group membership request"
* Tap 3-random name in "Pending memrship requests" block

#### Expected behavior
3-random name of the user in the bottom sheet

#### Actual behavior
"Unknown" user name in the bottom sheet
![](https://user-images.githubusercontent.com/67952253/102375204-873c1880-3fc2-11eb-998a-662f2274e49b.png)

- - -
* [ ] **ISSUE 10: Placeholder "Enter ENS or chat key" in "Edit group chat name" field**

_Steps:_

* Create a group chat
* Open Group info
* Tap Edit icon
* Remove name

#### Expected behavior
"Enter new topic"

#### Actual behavior
"Enter ENS username or chat key"

![](https://user-images.githubusercontent.com/67952253/102476984-0fbac780-405c-11eb-9732-337b333d9a90.png)

- - -
* [ ] **ISSUE 11: If status consists of 299 text characters and if add one emoji to it, then all the text will be deleted on iOS**

_Steps:_

* Open Status tab
* Tap `+` button
* Enter 299 text characters
* Enter one emoji

https://user-images.githubusercontent.com/67952253/105058954-12537700-5a77-11eb-84a3-bb3ff5c91b1c.mp4

- - -
* [ ] **ISSUE 12: Userpic changes its color every time when opening `Favorites` on the `Choose recipient` screen**

https://user-images.githubusercontent.com/67952253/106469789-0668ab80-64a0-11eb-888b-d5c8fd4db824.mp4

- - -
* [ ] **ISSUE 13: Chat suggestion appears on the `Home screen` if open the `Start new chat `screen, enter ENS and tap `X` button. Relevant only if there are no chats on the `Home screen`**

https://user-images.githubusercontent.com/67952253/106471008-7deb0a80-64a1-11eb-82d4-cf124a0c6c99.mp4

- - -
* [ ] **ISSUE 14: Self-mention with ENS with .stateofus.eth domain is resolved incorrectly or not resolved at all in the sent message.**

Sometimes the issue isn't reproducible on the first try.

_Reproduction:_ (needs clarification)

* Set ENS name with .stateofus.eth domain in Settings
* Join public chat
* Another User mentions you
* Send self-mention

<img width="372" alt="Screenshot 2021-03-10 at 15 20 43" src="https://user-images.githubusercontent.com/67952253/110643251-30567180-81b4-11eb-8ffc-052083d50eef.png">

- - -

* [ ] **ISSUE 15: After declining an invitation to a group chat user stops receiving messages in 1-1 chat and further group chats invitations. Relevant for repeatedly recovered multiaccounts only**

_Reproduction:_
Device 1:
- Recover multiaccount with seed phrase

Device 2:
- Recover multiaccount with seed phrase
- Send a message to Device 1 in 1-1 chat
- Add Device 1 to contacts
- Create a group chat with Device 1

Device 1:
- Decline invitation

Device 2:
- Send one more message in 1-1 chat

- - -

* [ ] **ISSUE 16: Nickname disappears on primary device after pairing and adding user to contacts on secondary device**

If set a nickname for some non-contact user before pairing on the primary device, then pair devices and add this user to contacts on the secondary device, the nickname will disappear on the primary device.

_Reproduction:_

Device 1:
- Set nickname to some non-contact user
- Sync with Device 2
- Open Profile of user with nickname

Device 2:
- Sync with Device 1
- Open Profile of user with nickname (nickname isn't visible on this device)
- Add user to contacts

Device 1:
- Check nickname (it disappeared at the same time the user was added to contacts)

https://user-images.githubusercontent.com/67952253/110658311-df4d7a00-81c1-11eb-84b6-5f66b35b7aa1.mp4

- - -

- [ ] **ISSUE 17: If shorten (edit) long message it will remain collapsed**

https://user-images.githubusercontent.com/67952253/146927903-c5038b95-5a4c-4d1c-8604-669b3d45183e.mp4

- - -

- [ ] **ISSUE 18: 1-1 chat with an invitation to the community is highlighted as new for sender**

**Comment by churik** [comment]
- [ ] **ISSUE 17: Issues in error handling when scanning QRs generated from universal links containing particular characters or upper case** https://github.com/status-im/status-react/issues/11116

**Comment by pavloburykh** [comment]
- [ ] **ISSUE 19: Profile page closes after opening on IOS device if it has been opened through avatar icon after longpress on message**. 

https://user-images.githubusercontent.com/97245802/158243208-27e4b2fa-c512-45ec-9188-7b2d53d051ec.mp4

**Comment by pavloburykh** [comment]
- [ ] **ISSUE 20: Implement user-friendly handling of an error 'execution reverted' which can occur during purchase**. 

Error is not reproducible every time. I reproduce it on my account with certain balance while trying to purchase some certain items with low price. 

**Steps:** 

1. Open Browser
2. Go to  opensea.io / cryptokitties.co
3. Try to purchase some collectible
4. When signing modal will appear - pay attention at the error then open network fee and see if Gas limit amount has been calculated

**Actual result:** Error 'execution reverted'. Gas limit amount field is empty. Bug does not reproduces every time and on every account. Ping me if you will need assistance with reproduction. 

**Expected result:** Gas limit amount field has calculated gas value

![photo_2022-08-10 15 06 19](https://user-images.githubusercontent.com/97245802/183896889-1d2f5f68-42a5-4168-a40f-50648f496824.jpeg)

https://user-images.githubusercontent.com/97245802/183240911-b7494828-55c6-477a-8d9c-3691ee9a79f6.mp4

According to @flexsurfer '`execution reverted` means there is an unexpected error in the contract, and it's not related to this issue '

So all we can do is to improve handling of such case in our UI.

**Comment by churik** [comment]
Obsolete and will be re-considered in the redesigned app. 
Added `recheck` label for easy finding that one in case of need - we should recheck these cases on new UI if the still make sense
